datamade/census
# Add 2019 for ACS 1- and 5-year estimates

## Description

See title. Closes #95.

## Testing instructions

- View the 1- and 5-year ACS pages and confirm 2019 is available:
  - https://www.census.gov/data/developers/data-sets/acs-1year.html
  - https://www.census.gov/data/developers/data-sets/acs-5year.html
